ingest/adash/cli/watch/__init__.py

```diff
@@ -148,7 +148,7 @@ def __init__(self, session, api_url):
             'Content-Length': str(size)
         }
         headers.update(self._headers)
-        upload = await self.post('file', params=params, headers=headers, data=br)
+        upload = await self.post('file', params=params, headers=headers, data=bits)
```

Comment:
Not sure what is going on here, my understanding is that data could be a file-like object, like the buffered reader being passed in?

Reply:
Yes, your understanding is correct. It seems like passing the BufferedReader would work just fine. But in practice our SVG images were getting corrupted, where the end of one file would sometimes bleed into the beginning of the next one. Reading the data out of the buffer ahead of time fixed this issue.